### PR TITLE
Alpha installerconfig

### PR DESCRIPTION
**Pull request by Wagnerp**
* Going script crazy! (Easier to build packages, with minimal config changes) ;)

### CODE REVIEW

File: Directory.Build.props [hunk 1]
```diff
@@ -1,27 +1,43 @@
...
-		<!--<CurrentDate>$([System.DateTime]::Now.ToString(ddHH))</CurrentDate>-->
+	<Choose>
+		<When Condition="'$(Configuration)' == 'Installer'">
+			<PropertyGroup>
+				<Minor>$([System.DateTime]::Now.ToString(MM))</Minor>
+				<Build>$([System.DateTime]::Now.get_DayOfYear().ToString())</Build>
 
-		<Minor>$([System.DateTime]::Now.ToString(yyMM))</Minor>
-		<Build>$([System.DateTime]::Now.get_DayOfYear().ToString())</Build>
+				<!--As WIX installer can only cope with build numbers <= 256, the yyMM layout won't work.
```
Comment (Smurf-IV) [inline Directory.Build.props:8]: This is not going to work, as 2212 will be the same as 2112

Comment (Smurf-IV) [reply Directory.Build.props:8]: Minor = yy
Release = MM
Build = doy

Or Minor = yy
Build = doy.

File: Source/Krypton Components/Directory.Build.props [hunk 2]
```diff
@@ -1,44 +1,83 @@
...
+	<Import Project="$([MSBuild]::GetPathOfFileAbove('Directory.Build.props', '$(MSBuildThisFileDirectory)../../'))" />
+
+	<PropertyGroup>
+		<SignAssembly>false</SignAssembly>
+		<AssemblyOriginatorKeyFile>StrongKrypton.snk</AssemblyOriginatorKeyFile>
+	</PropertyGroup>
+
+	<!--Handle AssemblyInfo-->
+	<PropertyGroup>
+		<GenerateAssemblyInfo>true</GenerateAssemblyInfo>
```
Comment (Smurf-IV) [inline Source/Krypton Components/Directory.Build.props:11]: Why have the spaces been replaced with tabs ?

File: Directory.Build.props (hunk 1 above)
Comment (Wagnerp) [reply Directory.Build.props:8]: Or the revision becomes doy i.e 6.22.01.1?

File: Source/Krypton Components/Directory.Build.props (hunk 2 above)
Comment (Wagnerp) [reply Source/Krypton Components/Directory.Build.props:11]: Might have happened when I was playing about

**Comment by Wagnerp** [comment]
@Smurf-IV Do we really need the `lite` stuff?

**Comment by Smurf-IV** [comment]
> @Smurf-IV Do we really need the `lite` stuff?

I do not think so any more.

File: Directory.Build.props (hunk 1 above)
Comment (Smurf-IV) [reply Directory.Build.props:8]: > Or the revision becomes doy i.e 6.22.01.1?

Yes, use all 4 number slots :-)

**Comment by Wagnerp** [comment]
@Smurf-IV A quick update, as there are now several `cmd` files, I've now moved them into a new `Script` folder along with modified `proj` files. I'm going to reconfigure `run.cmd` to streamline the process using numbered menus.

**Comment by Smurf-IV** [comment]
Is this ready to go now ?

**Comment by Wagnerp** [comment]
> Is this ready to go now ?

It is, though I need to rewrite `run.cmd` for a more 'streamlined' build experience.

**Pull request merged**